charts/stable/scalardl/templates/envoy/deployment.yaml

```
@@ -47,16 +47,7 @@ spec:
               path: /ready
               port: 9001
             failureThreshold: 3
-            initialDelaySeconds: 60
-            periodSeconds: 10
-            successThreshold: 1
-            timeoutSeconds: 1
-          readinessProbe:
-            httpGet:
-              path: /ready
-              port: 9001
-            failureThreshold: 3
-            initialDelaySeconds: 20
+            initialDelaySeconds: 30
```

After some investigation, I feel it should be way smaller since envoy pod almost starts up instantly.
```suggestion
            initialDelaySeconds: 10
```
Also, `minReadySeconds` can be 0 for the same reason. (I think it should start within 10 seconds in total since it is started, otherwise, there is something wrong)

I think the same configuration can be applied for scalar-ledger as well since it won't take that much time (usually a few seconds) even if JVM starts-up time is relatively slower than a simple process start-up.

@superbrothers The reason for the change is just to avoid the unnecessary time-wait. Will the above change could cause some issues potentially?

---

`InitialDelaySeconds` is generally set when there is some pre-processing that the server knows to do in order to provide the service. For example, "it will take time for the server to start" or "some data will be downloaded".

If `InitialDelaySeconds` is set to `10`, then with this setting the container will be killed if it cannot be served within 40s (`InitialDelaySeconds` + `periodSeconds` * `failureThreshold`). If the value is large, it will take longer to roll out.